apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2024-03-29T14:25:17Z"
-  build_hash: e8df4d5a4b86dea0e227786c2c3d213e5aeda97a
-  go_version: go1.22.0
-  version: v0.33.0
-api_directory_checksum: c67645b15db39980ba51ff6303c34c5aafc55a9e
+  build_date: "2024-04-07T05:45:26Z"
+  build_hash: c12a3d412eec1edac95eac3109e0a7cbc386ce43
+  go_version: go1.21.6
+  version: v0.33.0-6-gc12a3d4
+api_directory_checksum: 86b652bf0390d32e2720c12143f3c7631967bdcc
```

Comment:
Can we regenerate this PR using latest code-generator now?

Reply:
Done